lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
In my opinion it seems better to let the bindings module handle the finding/loading of the bindings file.

However you want to structure the path is fine as long as all of the build artifacts exist and the binary deployments still work.